annotation-web/src/main/java/eu/europeana/annotation/web/service/controller/jsonld/WebAnnotationAuxiliaryMethods.java

```diff
@@ -73,25 +76,32 @@
 
     @RequestMapping(value = "/annotations/deleted", method = RequestMethod.GET, produces = {
 	    HttpHeaders.CONTENT_TYPE_JSON_UTF8 })
-    @ApiOperation(value = "Get ids of deleted Annotations", nickname = "getDeletedAnnotationSet", response = java.lang.Void.class)
+    @ApiOperation(value = "Get ids of deleted Annotations", nickname = "getDeletedAnnotationSet", response = java.lang.Void.class,
+    		notes = "The afterDate and beforeDate parameters should have the format dd-mm-yyyy.")
     public ResponseEntity<String> getDeleted(
 	    @RequestParam(value = WebAnnotationFields.PARAM_WSKEY, required = false) String apiKey,
 	    @RequestParam(value = "motivation", required = false) String motivation,
 	    @RequestParam(value = "afterDate", required = false) String startDate,
-	    @RequestParam(value = "afterTimestamp", required = false) String startTimestamp, HttpServletRequest request)
+	    @RequestParam(value = "beforeDate", required = false) String stopDate,
+	    @RequestParam(value = "page", required = false, defaultValue = "0") int page,
+	    @RequestParam(value = "limit", required = false, defaultValue = "100") int limit,
+	    HttpServletRequest request)
 	    throws HttpException {
 
 	// SET DEFAULTS
 	verifyReadAccess(request);
 
 	MotivationTypes motivationType = validateMotivation(motivation);
```

Comment:
parsing and validation of the afterDate and BeforeDate should be performed here

Reply:
Done.